src/main/java/net/snowflake/ingest/streaming/internal/ParquetValueParser.java

```diff
@@ -59,14 +59,16 @@ static ParquetBufferValue parseColumnValueToParquet(
           AbstractRowBuffer.ColumnPhysicalType.valueOf(columnMetadata.getPhysicalType());
       switch (typeName) {
         case BOOLEAN:
-          int intValue = DataValidationUtil.validateAndParseBoolean(value);
+          int intValue =
+              DataValidationUtil.validateAndParseBoolean(columnMetadata.getName(), value);
```

Comment:
is this the display name as well?

Reply:
Yeah, according to https://github.com/snowflakedb/snowflake-ingest-java/blob/master/src/main/java/net/snowflake/ingest/streaming/internal/ArrowRowBuffer.java#L111 we set the columnMetadata.getName() as the displayName inside the RowBufferStat